### PR TITLE
Update numa cases for unavailable parm

### PR DESCRIPTION
**Pull request by hsmj1412**
Signed-off-by: hsmj1412 <ljx1412@gmail.com>

### CODE REVIEW

File: libvirt/tests/src/numa/numa_memory_migrate.py [hunk 1]
```diff
@@ -52,10 +52,14 @@ def mem_compare(used_node, left_node):
...
     node_list = host_numa_node.online_nodes
+    memory_list = host_numa_node.get_all_node_meminfo()
+    for nodenum in memory_list:
+        if int(memory_list[nodenum]["MemTotal"]) == 0 and nodenum in node_list:
+            node_list.remove(nodenum)
     logging.debug("host node list is %s", node_list)
     if len(node_list) < 2:
-        raise exceptions.TestSkipError("At least 2 numa nodes are needed on"
-                                       " host")
+        raise exceptions.TestSkipError("At least 2 useable numa nodes are"
```
Comment (dzhengfy) [inline libvirt/tests/src/numa/numa_memory_migrate.py]: Please change 'useable' to 'usable' and  change 'raise exceptions.TestSkipError' to 'test.cancel'

**Comment by hsmj1412** [comment]
@dzhengfy done

File: libvirt/tests/src/numa/numa_memory_migrate.py [hunk 2]
```diff
@@ -46,16 +62,21 @@ def mem_compare(used_node, left_node):
...
     node_list = host_numa_node.online_nodes
+    memory_list = host_numa_node.get_all_node_meminfo()
+    for nodenum in memory_list:
+        if int(memory_list[nodenum]["MemTotal"]) == 0 and nodenum in node_list:
+            node_list.remove(nodenum)
     logging.debug("host node list is %s", node_list)
     if len(node_list) < 2:
-        raise exceptions.TestSkipError("At least 2 numa nodes are needed on"
-                                       " host")
+        raise test.cancel("At least 2 usable numa nodes are"
```
Comment (waynesun09) [inline libvirt/tests/src/numa/numa_memory_migrate.py]: test.cancel will raise error, no need use raise before it

File: libvirt/tests/src/numa/numa_memory.py [hunk 3]
```diff
@@ -28,6 +28,22 @@ def _logger(line):
         """
         numad_log.append(line)
 
+    def cpu_adapter_mod(par_list, std_list):
```
Comment (waynesun09) [inline libvirt/tests/src/numa/numa_memory.py]: for this common function, pls add it in virttest

**Comment by waynesun09** [comment]
@hsmj1412 your branch is not clean, 2 commits in this PR now

**Comment by hsmj1412** [comment]
[https://github.com/avocado-framework/avocado-vt/pull/1134](url)

**Comment by hsmj1412** [comment]
This PR will depend on https://github.com/avocado-framework/avocado-vt/pull/1134

**Pull request closed without merging**